### PR TITLE
[CCXDEV-14176] GH actions: add konflux PR automerging

### PR DESCRIPTION
**Pull request by matysek**
# Description

[CCXDEV-14176] GH actions: add konflux PR automerging

## Type of change

- Configuration update

### CODE REVIEW

File: .github/workflows/bots-automerge.yml [hunk 1]
```diff
@@ -3,38 +3,32 @@ name: Bots auto-merge
 on: pull_request
 
 # This section adds write permissions to the secrets.GITHUB_TOKEN. Default is just read
-#permissions:
-  #contents: write
-  #pull-requests: write
+permissions:
```
Comment (epapbak) [inline .github/workflows/bots-automerge.yml:6]: why is this needed?

Comment (matysek) [reply .github/workflows/bots-automerge.yml:6]: default permissions are mostly read-only for the github_token. This allows to set approvals on pull requests for GH actions and enable automerging.

File: .github/workflows/bots-automerge.yml [hunk 2]
```diff
@@ -3,38 +3,32 @@ name: Bots auto-merge
...
+      - name: InsightsDroid approves the PR
+        run: gh pr review --approve "$PR_URL"
+        env:
+          PR_URL: ${{github.event.pull_request.html_url}}
+          GH_TOKEN: ${{secrets.INSIGHTSDROID_TOKEN}}
+      - name: Enable auto-merge for Dependabot PRs
+        run: gh pr merge --auto --merge "$PR_URL"
+        env:
+          PR_URL: ${{github.event.pull_request.html_url}}
+          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```
Comment (juandspy) [inline .github/workflows/bots-automerge.yml]: why not `github.token`?

Comment (matysek) [reply .github/workflows/bots-automerge.yml]: sorry, forgot to do push of the changes.

**Review by juandspy — APPROVED** [review]
